tests/PHPStan/Analyser/data/call-user-func-array.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace CallUserFuncArray;
+
+use function PHPStan\Testing\assertType;
+
+function fun(): int
+{
+	return 3;
+}
+
+function fun3($i, $x, $y): int
```

Comment:
```suggestion
function fun3(int $i, string $x, bool $y): int
```

If we added types to the parameters in this test fixture, could we verify an `*ERROR*` type?

```
assertType('*ERROR*', call_user_func_array('CallUserFuncArray\fun3', [1 ,2 ,3]));
assertType('int', call_user_func_array('CallUserFuncArray\fun3', [1 ,'2' ,false]));
```

Reply:
I did a quick test locally, and it _does not_ come back as an error type. So the parameters not not checked.